app/src/main/java/swati4star/createpdf/fragment/ImageToPdfFragment.java

```diff
@@ -505,6 +510,32 @@ private void saveImagesInGrayscale() {
         }
     }
 
+    /**
+     * Returns the last file name obtained from filepath
+     */
+    private String getLastFileName(ArrayList<String> filesPath) {
```

Comment:
there are already a lot of functions in this file.
Can we move this and next function to FileUtils?

Reply:
yes, moved to FileUtils.